src/components/Home.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+class Promo extends React.Component {
+  constructor () {
+    super();
+
+    this.state = {
+      chief: ''
+    };
+  }
+
+  componentDidMount () {
+    const studentsPromo = this.props.promo.students;
+    let chiefPromo = null;
+    this.props.chiefs.forEach(function (chief) {
+      if (studentsPromo.includes(parseInt(chief.id, 10))) {
+        chiefPromo = chief;
+      }
+    });
+    this.setState({
+      chief: chiefPromo
+    });
+  }
+
+  render () {
+    if (this.state.chief != null) {
```

Comment:
`const chiefContent = this.props.chief ? (<div className="chief"/>) : (<div />)`